docs/pt/docs/deployment/index.md

```diff
@@ -0,0 +1,7 @@
+# Implantação - Introdução
```

Comment:
I think `Deploy` is often translated as `implantação`.

Reply:
I've seen `Deploy` translated as `implantação` and `Implementação`, but I believe that the first one makes more sense because we're inserting it in the cloud. 

I believe that `Implementação` would be a synonym to `implantação` in our context.

I will leave here some resources that helped me to pick `implantação`:
[Comparative between the two words](https://www.sinonimos.com.br/implantar/)
[Implantar synonyms/meaning](https://www.sinonimos.com.br/implantar/)
[implementar synonyms/meaning](https://www.sinonimos.com.br/implementar/)

Let me know what you guys think and if I should keep it or switch to `Implementação`.